src/components/phyx/PhyxView.vue

```diff
@@ -65,14 +65,28 @@
                 @change="$store.commit('setDefaultNomenCodeURI', { defaultNomenclaturalCodeURI: $event.target.value })"
               >
                 <option
-                  v-for="(nomenCode, nomenCodeIndex) of nomenCodes"
-                  :value="nomenCode.uri"
+                  v-for="nomenCode of nomenCodes"
+                  :value="nomenCode.iri"
                 >
                   {{ nomenCode.label }}
                 </option>
               </select>
             </div>
           </div>
+
+          <!-- Checkbox for permission to save this information to browser -->
+          <div class="form-group row">
+            <div class="col-md-2">&nbsp;</div>
+            <div class="col-md-10">
+              <input type="checkbox"
+                     :checked="cookieCheckbox"
+                     @click="$store.commit('toggleCookieAllowed')"
+              />
+              Click this checkbox if you would like Klados to save your curator information and
+              default nomenclatural code as a browser cookie for the next month, or uncheck it to
```

Comment:
```suggestion
              default nomenclatural code as a browser cookie (currently until the next month). If changed to unchecked, 
```

Reply:
I've made this explicit as "(currently for thirty days)." in ddac2cd. However, we can easily change that to one month if that is preferable (as per https://www.npmjs.com/package/vue-cookies).